ghost-cat-app/src/app/View/search-page/search-page.component.ts

```diff
@@ -37,8 +39,47 @@ export class SearchPageComponent implements OnInit {
       new CameraLocation('site006', 40.77956, -110.67389),
       new CameraLocation('site008', 40.77956, -110.77389),
     ];
+    for(let i = 0; i < this.cameraTrapsSelected.length; i++){
+      alert("Current Cameras selected " + this.cameraTrapsSelected[i] + "\n");
+    }
   }
 
+  checkedCameras(value) {
+    //this function is called to select or deselect individual camera check boxes and adjust the final search array(cameraTrapsSelected) accordingly
+    if ((<HTMLInputElement>document.getElementById(value)).checked === true) {
+        this.cameraTrapsSelected.push(value);
+        (<HTMLInputElement>document.getElementById("Remove_All")).checked = false;
+        (<HTMLInputElement>document.getElementById("Select_All")).checked = false;
+    }
+    else if ((<HTMLInputElement>document.getElementById(value)).checked === false) {
+        let indexx = this.cameraTrapsSelected.indexOf(value);
+        this.cameraTrapsSelected.splice(indexx,1);
+        (<HTMLInputElement>document.getElementById("Remove_All")).checked = false;
+        (<HTMLInputElement>document.getElementById("Select_All")).checked = false;
+    }
+  }
+
+  bulk(a) {
+    //this function is called to select or deselect all camera check boxes and adjust the final search array(cameraTrapsSelected) accordingly
+    if (a == "all") {
+        console.log("yes all");
```

Comment:
Like above, we probably want to get rid of this `console.log` before adding it to the final website